acceptance-tests/src/test/java/uk/nhs/digital/ps/test/acceptance/pages/LoginPage.java

```diff
@@ -0,0 +1,63 @@
+package uk.nhs.digital.ps.test.acceptance.pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.WebElement;
+import uk.nhs.digital.ps.test.acceptance.webdriver.WebDriverProvider;
+
+public class LoginPage extends AbstractPage {
+
+
+    public LoginPage(final WebDriverProvider webDriverProvider) {
+        super(webDriverProvider);
+    }
+
+    public void open() {
+        getWebDriver().get(URL);
+    }
+
+    public void loginWith(final String username, final String password) {
+        getUsernameField().sendKeys(username);
+        getPasswordField().sendKeys(password);
+        getLoginButton().click();
+    }
+
+    private WebElement getLoginButton() {
+        return getWebDriver().findElement(By.name("p::submit"));
+    }
+
+    private WebElement getPasswordField() {
+        return getWebDriver().findElement(By.name("password"));
+    }
+
+    private WebElement getUsernameField() {
+        return getWebDriver().findElement(By.name("username"));
+    }
+
+    public boolean hasLoginErrorMessage(String expectedErrorMessage){
```

Comment:
Is this used? Looks like it's not.

Reply:
Indeed, I left it behind by mistake having done refactoring that rendered this method unused. Gone now.